packages/components/src/components/domainproperties/actions.ts

```diff
@@ -118,6 +118,22 @@ export function fetchDomain(domainId: number, schemaName: string, queryName: str
     });
 }
 
+export function fetchQueryMetadata(domainId: number, schemaName: string, queryName: string): Promise<DomainDesign> {
```

Comment:
See comments related to this function from other PR review notes.

Reply:
Removed this function from here.